contracts/test/integration/IntegrationTestPriceBoundPSMUSDC.t.sol

```diff
@@ -0,0 +1,332 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
```

Comment:
add a test case where I set the oracle to be a new oracle contract with a random value as the current oracle price, then make sure things work as expected